api/jobs.go

```diff
@@ -82,3 +82,36 @@ func (api *JobStoreAPI) GetJobHandler(ctx context.Context) http.HandlerFunc {
 		}
 	}
 }
+
+//GetJobsHandler returns a function that gets a list of existing Job resources, from the Job Store, sorted by
+//their values of last_updated time (ascending).
+func (api *JobStoreAPI) GetJobsHandler(ctx context.Context) http.HandlerFunc {
+	log.Event(ctx, "Entering GetJobsHandler function, which returns a list of existing Job resources held in the JobStore.", log.INFO)
```

Comment:
Isn't this log hit when the handler is created at startup rather than when entered?

Reply:
In terms of when the _handler_ is entered, I'm sure you're correct. But that log is the first line of the GetJobsHandler _function_ so whenever that function is called then the log will be hit (and that can only be when the function has been entered). I'm happy to move it somewhere else if you prefer. However, I would have to do the same for the other API functions as they all have logging in a similar place.